Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<script>
+    (function() {
+        var recoClient{{ recommendationId }} = new EzRecoRestClient({
```

Comment:
you don't need to generate unique variables names since you are in an anonymous function. But actually, given how the client and the *renderer* are used, instead of defining 2 classes, I would define 2 functions instead (`fetchRecommendations(config,  callback)` and `renderRecommendations(config)` for instance)